modules/vsphere-problem-detector-viewing-events.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies
+//
+// * storage/vsphere-problem-detector-operator.adoc
+
+[id="vsphere-problem-detector-operator-view-events_{context}"]
+= Viewing the events from the vSphere problem detector
+
+After the vSphere Problem Detector Operator runs and performs the configuration checks, it creates events that can be viewed from the command line or from the monitoring user interface.
```

Comment:
You could replace "the monitoring user interface" with "the {product-title} web console" and then provide exact steps on how to access the appropriate UI page in the procedure.